includes/definitions/discovery/zynos.yaml

```diff
@@ -104,25 +180,7 @@ modules:
                     value: voltageCurValue
                     snmp_flags: ['-OteQUs', '-Pu'] # workaround for underscores in mib
                     num_oid: '.1.3.6.1.4.1.890.1.5.8.47.8.3.1.2.{{ $index }}'
+                    divisor: 1000
                     descr: '{{ $voltageNominalValue }}mV ({{ $voltageDescr }})'
                     index: 'voltageCurValue.{{ $index }}'
                     low_limit: voltageLowThresh
-                -
-                    oid: transceiverDdmInfoTable
```

Comment:
same as above

Reply:
Added back so I will resolve this conversation